### ISSUE TITLE
XML tags missing in 04_Separating_Data_and_Instructions

### ISSUE THREAD

**Issue by chengxuz** [anthropics/prompt-eng-interactive-tutorial#6]
No tags were actually added in the cells about adding XML tags, and the output was the same as the previous cell, which was incorrect.

**Comment by chengxuz** [comment]
BTW, all XML tags in the prompts seem weird in some way; there will be a blank in front of the `tags`, but no actual example tags are provided.

Also, in the "09_Complex_Prompts_from_Scratch", the `TASK_DESCRIPTION ` in the Legal example should be a `f` string to include the actual question.

**Comment by francis-beeson-sage** [comment]
Yes, it seems some parsing was used for moving string literals from one part of the repo to this one, and XML tags were removed. I   had to copy the literals from the spredsheet "Anthropic's Prompt Engineering Interactive Tutorial [PUBLIC ACCESS]" in order to make it work. Obviously, it means Anthropic either never ran the code in these notebooks, or didn't bother to check the results. This reflects poorly on Anthropic's attitude towards quality, though I'm not surprised given their over-indexing on DEI. 

It would be nice to see this corrected, as I was planning to share this tutorial with one of my client's team of IC working on my project, but then it would reflect poorly on me suggesting to them a tutorial that is defective. It's just easier for me to do what my gut tells me which is to change LLM providers.

I'm happy to share my files if requested, but I ran all tutorials in proper `.py` files, as I'm allergic to jupyter notebooks.

**Comment by ijindal1** [comment]
XML tags in the prompt aren't missing. Github isn't rendering them properly.

Attaching a screenshot from the Google Collab notebook

<img width="986" alt="Screenshot 2024-08-07 at 5 26 58 PM" src="https://github.com/user-attachments/assets/52441eeb-d6c3-41b1-aaa4-71435db11bdb">

**Comment by abdelmalek13** [comment]
It seems its missing on Github notebook viewer

<img width="1558" height="778" alt="Image" src="https://github.com/user-attachments/assets/b809bcb9-f09c-4784-b6d6-c8c0ecfd7ebf" />
